### ISSUE TITLE
how to add my own tags?

### ISSUE THREAD

**Issue by losaye** [starik222/BooruDatasetTagManager#26]
try to edit the csv file in tags directory, but error always occured

**Comment by losaye** [comment]
> For example, the quality.txt file has been added to the tags folder. You can add your own tags to this file, or create new text files with your own list of tags. For example, create a MyTags.txt file and add new tags line by line to it. When launched, the program will create a new database of tags, including those you added.

Thx for explanation. I repeated several times, neither created a new txt file nor edited the quality.txt can stop throwing out the same error, as shown below (translated by machine):

```
Exception text ******
System.ArgumentOutOfRangeException: Index out of range. Must be non-negative and less than the collection size.
Parameter name: index
   在 System.ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument argument, ExceptionResource resource)
   在 BooruDatasetTagManager.TagsDB.AddTag(String tag, Int32 count, Boolean isAlias, String parent)
   在 BooruDatasetTagManager.TagsDB.LoadFromCSVFile(String fPath, Boolean append)
   在 BooruDatasetTagManager.TagsDB.LoadCSVFromDir(String dir)
   在 BooruDatasetTagManager.Program.<><Main>c.b__0_1()
   在 System.Threading.Tasks.Task.Execute()
--- at the end of the stack trace at the previous position where the exception was thrown---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 BooruDatasetTagManager.Program.<>c__DisplayClass0_0. <<Main>b__0>d.MoveNext()
--- at the end of the stack trace at the previous position where the exception was thrown---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
```

**Comment by starik222** [comment]
Thank you, there is indeed such a mistake.

**Comment by starik222** [comment]
Fixed in [1.6.4](https://github.com/starik222/BooruDatasetTagManager/releases/tag/v1.6.4)
